_overviews/scala3-book/taste-objects.md

```diff
@@ -19,27 +19,43 @@ Objects have several uses:
   In this situation, that class is also called a _companion class_.
 - They’re used to implement traits to create _modules_.
 
-
-
 ## “Utility” methods
 
 Because an `object` is a Singleton, its methods can be accessed like `static` methods in a Java class.
 For example, this `StringUtils` object contains a small collection of string-related methods:
 
+
+{% tabs object_1 class=tabs-scala-version %}
+{% tab 'Scala 2' for=object_1 %}
+```scala
+object StringUtils {
+  def isNullOrEmpty(s: String): Boolean = s == null || s.trim.isEmpty
+  def leftTrim(s: String): String = s.replaceAll("^\\s+", "")
+  def rightTrim(s: String): String = s.replaceAll("\\s+$", "")
+}
+```
+{% endtab %}
+
+{% tab 'Scala 3' for=object_1 %}
 ```scala
 object StringUtils:
   def isNullOrEmpty(s: String): Boolean = s == null || s.trim.isEmpty
   def leftTrim(s: String): String = s.replaceAll("^\\s+", "")
   def rightTrim(s: String): String = s.replaceAll("\\s+$", "")
 ```
+{% endtab %}
+{% endtabs %}
 
 Because `StringUtils` is a singleton, its methods can be called directly on the object:
 
+{% tabs object_2 class=tabs-scala-version %}
```

Comment:
`class=tabs-scala-version` should not be used for "Scala 2 and 3" tabs. Soon I am adding assertions to make CI fail for this
```suggestion
{% tabs object_2 %}
```